src/components/RadioButtonGroup/index.js

```diff
@@ -17,6 +19,20 @@ class RadioButtonGroup extends Component {
         super(props);
         this.errorId = uniqueId('error-message');
         this.groupNameId = props.name || uniqueId('options');
+        this.handleChange = this.handleChange.bind(this);
+        this.isMarkerActive = this.isMarkerActive.bind(this);
+        this.optionsRefs = this.generateRefsForOptions();
+
+        this.state = {
+            markerLeft: 0,
+            markerWidth: 0,
+        };
+    }
+
+    componentDidMount() {
+        setTimeout(() => {
+            this.updateMarker();
+        }, 0);
```

Comment:
this is not needed:
you can simply use: ```this.updateMarker();```
remember we need to avoid setTimeout

Reply:
Had to set the settimeout cos atthis point dom has not set the styles entirely and the values of left and witdh are wrong.. it happends on chrome and safari